common/logger/src/struct_log.rs

```diff
@@ -16,7 +16,7 @@ use std::{
     fmt::Display,
     fs::{File, OpenOptions},
     io,
-    io::Write as IoWrite,
+    io::{Write as IoWrite, Write},
```

Comment:
Why does `Write` need to be brought in twice?

Reply:
hmm woops